src/mmw/js/src/modeling/models.js

```diff
@@ -497,7 +497,8 @@ var ScenarioModel = Backbone.Model.extend({
             // Makeshift locking mechanism to prevent double saves.
             this.saveCalled = true;
         }
-        this.save().fail(function() {
+        // Save silently so server values don't trigger reload
+        this.save(null, { silent: true }).fail(function() {
```

Comment:
Formatting-wise, I prefer the second chain beginning its own line so it stands out a bit more.

```javascript
this.save(null, {...})
    .fail(function() {});
```